From 121b9648f846d900e67818869974ee82046e9b25 Mon Sep 17 00:00:00 2001
From: Zhenyu Wang <zhenyuw@linux.intel.com>
Date: Fri, 04 Sep 2009 01:24:23 +0000
Subject: libdrm_intel: add new pci ids

New ids for G41, Clarkdale and Arrandale.
Make sure we don't need to count fence also on new chips.

Signed-off-by: Zhenyu Wang <zhenyuw@linux.intel.com>
---
diff --git a/libdrm/intel/intel_chipset.h b/libdrm/intel/intel_chipset.h
index 0b3af02..26bc585 100644
--- a/libdrm/intel/intel_chipset.h
+++ b/libdrm/intel/intel_chipset.h
@@ -48,7 +48,10 @@
                        (dev)->pci_device == 0x2A42 || \
                        (dev)->pci_device == 0x2E02 || \
                        (dev)->pci_device == 0x2E12 || \
-                       (dev)->pci_device == 0x2E22)
+                       (dev)->pci_device == 0x2E22 || \
+                       (dev)->pci_device == 0x2E32 || \
+                       (dev)->pci_device == 0x0042 || \
+                       (dev)->pci_device == 0x0046)
 
 #define IS_I965GM(dev) ((dev)->pci_device == 0x2A02)
 
--
cgit v0.8.2
